features/support/env.rb

```diff
@@ -24,7 +26,17 @@
 Selenium::WebDriver::Firefox::Binary.path = ENV["FIREFOX_BINARY_PATH"] || Selenium::WebDriver::Firefox::Binary.path
 
 Capybara.register_driver :selenium do |app|
-  Capybara::Selenium::Driver.new(app, browser: :firefox)
+  profile = Selenium::WebDriver::Firefox::Profile.new
+  # Set the download directory to "~/diaspora/tmp/downloads"
+  profile['browser.download.dir'] = DownloadHelpers::PATH.to_s
+  # Save the file instead of opening it
+  profile['browser.download.folderList'] = 2
+  # Hide the download Manager
+  profile['browser.download.manager.showWhenStarting']=false
+  # Suppress "open with" dialog for zipped files only
+  profile['browser.helperApps.neverAsk.saveToDisk'] = 'application/zip'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.